Search/Exception/MetadataNotFoundException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Search\Exception;
+
+class MetadataNotFoundException extends \RuntimeException
```

Comment:
Well, if you are throwing the exception in different places with the same meaning then you have a [SPOT](http://en.wikipedia.org/wiki/Single_Point_of_Truth) problem, if you are throwing the same exception but with a different context, then the message should be different in each case.